src/core/tour/TourManager.ttslua

```diff
@@ -110,10 +110,15 @@ do
   ---@param playerColor Player color to show the current card for
   internal.showCurrentCard = function(playerColor)
     internal.updateCardDisplay(playerColor)
-    local hook = getObjectFromGUID(tourState[playerColor].cameraHookGuid)
-    hook.setPositionSmooth(HOOK_CAMERA_HOME, false, false)
-    local delay = 0.5
+    local delay = 0
     local cardIndex = tourState[playerColor].currentCardIndex
+    local hook = getObjectFromGUID(tourState[playerColor].cameraHookGuid)
+--    local centerCamera = TOUR_SCRIPT[cardIndex].skipCentering and false or true
```

Comment:
Is this meant to stay as comment?

Reply:
Nope, should have been cleaned up.  Removed.